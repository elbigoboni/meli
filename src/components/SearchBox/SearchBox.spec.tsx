import React from "react";
import { fireEvent, render, screen } from "@testing-library/react";
import { SearchBox } from ".";
import { MemoryRouter, useHistory } from "react-router-dom";

describe("<SearchBox />", () => {
  beforeEach(() => {
    render(<SearchBox />, { wrapper: MemoryRouter });
  });
  describe("Render", () => {
    test("should render correctly", () => {
      const { container } = render(<SearchBox />, { wrapper: MemoryRouter });
      expect(container).toMatchSnapshot();
    });
  });
});
