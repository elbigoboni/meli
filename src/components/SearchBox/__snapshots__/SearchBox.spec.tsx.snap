// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBox /> Render should render correctly 1`] = `
<div>
  <form
    class="sc-bdfBwQ ggUMEn"
    data-testid="search box"
  >
    <div
      class="sc-hKgILt gFyfpC"
    >
      <input
        class="sc-gsTCUz iPwkQd"
        data-testid="search field"
        placeholder="Nunca dejes de buscar"
        type="text"
        value=""
      />
      <button
        class="sc-dlfnbm daFmpW"
      >
        GO
      </button>
    </div>
  </form>
</div>
`;
